README.md

```diff
@@ -10,7 +10,7 @@ Travis CI (Python 3): [![Build Status](https://travis-ci.org/gem/oq-engine.svg?b
 
 ### Current stable
 
-Current stable version is the **OpenQuake Engine 2.0** 'Aki'. The documentation is available at https://github.com/gem/oq-engine/tree/engine-2.0#openquake-engine.
+Current stable version is the **OpenQuake Engine 2.1** 'Benioff'. The documentation is available at https://github.com/gem/oq-engine/tree/engine-2.1#openquake-engine.
```

Comment:
In the `engine-2.1` branch `README.md` we'll need then to put

## OpenQuake Engine version 2.1 (Benioff)

Starting from OpenQuake version 2.0 we have introduced a "code name" to honour earthquake scientists.

The code name for version 2.1 is **Benioff**, in memory of [Hugo Benioff](https://en.wikipedia.org/wiki/Hugo_Benioff).